explore-analyze/visualize/charts/area-charts.md

```diff
@@ -0,0 +1,168 @@
+
+Use stacking to show how categories contribute to a total over time.
+
+1. Create an area chart with a time-based **Horizontal axis**.
+2. Break down the series by a categorical field, for example, `agent.keyword`, `response.keyword`.
+   You can set the area chart stack mode to:
+   - **Stacked** — Show cumulative totals and category contributions.
+
+     ![Example Lens area chart stacked mode](../../images/kibana-area-stacked.png " =70%")
+
+   - **Percentage (100%)** — Normalizes each timestamp to 100% to emphasize shares rather than magnitudes.
+
+     ![Example Lens area chart percentage mode](../../images/kibana-area-percentage.png " =70%")
+
+4. Optionally, set **Rank by** for the breakdown dimension to control stacking order.
+
+### Compare current versus previous period with time shift [area-timeshift]
+
+In Area charts, you can enable time shift to compare different periods and identify deltas.
+
+![Example Lens area chart compare periods](../../images/kibana-area-compare-periods.png)
+
+1. Create an area chart with a time-based **Horizontal axis** and your main metric on the **Vertical axis**, for example, `bytes`.
+2. From the three-dot menu in the upper-left of the layer panel, select **Duplicate layer**.
+3. From the duplicated layer, open the **Advanced** of the **Vertical axis** and set **Time shift** to `1w`. 
+   Check also [Compare differences over time](../lens.md#compare-data-with-time-offsets) for more details.
+4. Use a different color and set **Stacking** to **None** so areas overlay rather than stack.
+5. Update the legend to show “Current” and “Previous”.
+
+::::{tip}
+You can also compute the relative change using a formula, for example:  
+`(average(bytes) - average(bytes, shift='1w')) / average(bytes, shift='1w')`
+:::: 
+
+## Area chart settings [area-chart-settings]
+
+Customize your area chart to match the information you need and how you want it displayed.
+
+### Horizontal axis settings [horizontal-axis-settings]
+
+**Data**
+:   **Functions**: Allow you to group your data. For example, you can use `Date histogram` to group data points into time-based buckets, or `Filters` to divide values into predefined subsets.
+:   **Fields**: Determine which field from your data will be used for the horizontal axis.
+:   **Minimum interval**: Determine the smallest time bucket that your data will be grouped into (for example, seconds, minutes, hours, days).
+
+**Appearance**
+:   **Name**: By default, the chart uses the function or formula as title. It's a best practice to customize this with a meaningful title.
+
+### Vertical axis settings [vertical-axis-settings]
+
+**Data**
+:   To represent the metrics or values you want to visualize, you can use quick functions like `Average`, `Count`, `Percentile`, `Counter rate`, or create custom calculations with formulas. Refer to [](/explore-analyze/visualize/lens.md#lens-formulas) for examples, or to the {icon}`documentation` **Formula reference** available from Lens.
```

Comment:
Where is the Formula reference? I can't find it on the page and searching for it doesn't bring up any results.

<img width="1405" height="983" alt="Image" src="https://github.com/user-attachments/assets/0cbc1fa6-45d5-4856-9dc0-4530fadcbc2a" />